papers/P2988/mybiblio.bib

```diff
@@ -58,3 +58,12 @@ @misc{rawgithu58:online
 year = {},
 note = {(Accessed on 08/14/2024)}
 }
+
+@misc{The_Beman_Project_beman_optional26,
+author = {The Beman Project},
+license = {Apache-2.0},
+title = {{beman.optional26}},
+howpublished = {\url{https://github.com/bemanproject/optional26}},
```

Comment:
Same reminder here about renaming. Feel free to ignore it. Just trying to help for Austria.